docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The Dafny Language and Software Verification System
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+---
+<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+<script type="text/x-mathjax-config">
+        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ["\\(","\\)"]], displayMath: [ ["$$","$$"], ["\\[","\\]"] ]
+        }});
+</script>
+
+<link rel="stylesheet" href="assets/main.css">
+
+**Quick Links:**
+- [Dafny Reference Manual and User Guide](DafnyRef/DafnyRef)
+- [Index to Dafny Resources for Users](toc)
+- [Dafny GitHub project (for developers of the Dafny tools themselves)](https://github.com/dafny-lang/dafny)
```

Comment:
This may be  a good spot to link to the VS code extension.

Reply:
Added the link